app/views/organisations/_organisation_list.html.erb

```diff
@@ -0,0 +1,27 @@
+<% content_for :title, title %>
+
+<%= govuk_table do |table| %>
+  <%= table.caption(size: "s", classes: %w[govuk-!-text-align-left govuk-!-margin-top-4 govuk-!-margin-bottom-4]) do |caption| %>
+    <span class="govuk-!-margin-right-4">
+      <strong><%= @pagy.count %></strong><span style="font-weight: normal"> total <%= title.downcase %></span>
+    </span>
+  <% end %>
+  <%= table.head do |head| %>
+    <%= head.row do |row| %>
+      <% row.cell(header: true, text: "Name") %>
+      <% row.cell(header: true, text: "Registration number") %>
```

Comment:
What is registration number here? Is that just ID?

Reply:
`organisation.housing_registration_no` we've got this column in out schema so I assume that's what it is

Reply:
> Is there a ticket to add a "Create Organisation" button for support users?

Hmm, not that I know of. Was not in AC for this one either tho